### DISCUSSION TITLE
relationship between the PHP Imagick class and the magick command

### DISCUSSION THREAD

**Discussion by cmette** [ImageMagick/ImageMagick#5198] [Consulting]
Hello to the round,

can someone explain me in more detail the relationship between the PHP Imagick class and the magick command of the command line tools? (I am using PHP 8.1.6 with ImageMagick 7.1.0.18).

I want to use the commands from the example here:
https://imagemagick.org/script/convex-hull.php

...translate into PHP code. In general there should be an equavalent call to the class Imagick - or not?

$imagick = new \Imagick();
$imagick->convexHull() ?? 

but this is not the case. 

Is there any way to translate the Examples from https://imagemagick.org/script/ into PHP code?

Many greetings.
Theo

**Comment by snibgo** [comment]
ImageMagick is a raster image processor. It has a command line interface. It has other interfaces, such as C and C++. This forum is for ImageMagick.

IMagick is a separate product. It is not maintained by the same people as ImageMagick.

I have no experience of IMagick.

**Comment by cmette** [comment]
Thanks for the quick reply! I will investigate further at Imagick...

**Comment by chetan-ids** [comment]
Hello,
Color emojis charcter not working in Imagick image displaying black&white emoji. I am using this font "OpenSansEmoji.ttf" also tried 'NotoEmoji-Regular.ttf', 'OpenMoji-Color.ttf', Can someone please let me know?
